# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: serkan@gentoo.org-20090501130741-o8b6blizbhjzjj6h
# target_branch: ../mainline/
# testament_sha1: 2fc183c2dc2d25479ed1d57daac41eca83348a6a
# timestamp: 2009-05-01 16:07:55 +0300
# base_revision_id: andrew@operationaldynamics.com-20090501052544-\
#   ulidr3e8b5emqalu
# 
# Begin patch
=== modified file 'build/faster'
--- build/faster	2009-03-19 04:28:31 +0000
+++ build/faster	2009-05-01 13:07:41 +0000
@@ -340,6 +340,7 @@
 		cmd += " -classpath " + classpath
 	if sourcepath:
 		cmd += " -sourcepath " + sourcepath
+	cmd += " -encoding UTF-8"
 	cmd += " " + " ".join(sources)
 
 	blurb = "\n\t".join(sources)
@@ -651,6 +652,7 @@
 	cmd += "-stylesheetfile src/bindings/stylesheet.css "
 	cmd += "-overview src/bindings/overview.html "
 	cmd += "-sourcepath src/bindings "
+	cmd += "-encoding UTF-8 "
 	cmd += "-subpackages org "
 	cmd += "-exclude org.freedesktop.bindings "
 	cmd += "src/bindings/org/freedesktop/bindings/Time.java "

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWSA2IHoAA7tfgCAwUWv//3//
//C////wYAc/AAAAUAAAAANREaNBoMgBpoDRkANGgwmjTEAxxkyaGIxNGARgJhAGAmmjTI0Axxky
aGIxNGARgJhAGAmmjTI0AxxkyaGIxNGARgJhAGAmmjTI0AxxkyaGIxNGARgJhAGAmmjTI0AwVSIm
TRiDQmIAmCaTDUMptPSbRNMiPI0lhFC5qatNDPr3lr54vFKXBFbS/b3Xvx+tqMhJFKTN9+GjRPjt
14pR++ebJEGRj5Nakq01XqqKcKKNnQ96uPg3Q4FaRKUpfz5NKP5fapYJGF9dCe2hY/A9hQ/RHz5k
v6zsbWxNLO/qbma8ihjUpvrI/pIvWK1A9j7Xdzkn1zpepNwrkm6jVLmlr550eTsv8kIxYuVvUwhq
VQ+7DPQ9P7ePoIPajuRoOh0Ocv7SUIdsI/U4ufPvOpg3Kh5UWN8SozOc9QjQWmbV8nJd/xDn0bJp
znyc+lj3LiKHsT1uZI2AxsZtShDphG5l3EqZIRMrtK0kSKax9YRU4KkIuqf6XFVjQwxqGZk63q8/
9Xvc6k0XJRgkkoMM4t/J+zOdLPvaSlcShCxNakVttuGuyRerhC6Ly2EZ1GO+plWt1a77a0lrFIzu
+EYXMGGhkvUxEfdTrBw5zSuZE1hQmiZ05GnOuo5MrgiyDLPhoZnWzwjMZqWWe+uELZ4p3aIwZ64v
U72tU/yuMqS7G1YXXNpRY2mejRpZTOmaTEJs65TCLllF+QvVLFbQsbLe2GBPMyrlzK2x/dE7pcRI
zaIIy+vgXMxiMkaiTbavyYqtK44ydjHOvYxwyxTkbF/Mx3xabDn2c07IdjJWM7N0jmz5rYq1kXsi
ZRbJRlpzSFLBKpKWKQ5LG7i45NrAoywjI5iqulLVKclTbwsty9x8eJjVP9JKyilrPMNkImkoYNba
n7cLFWs5GvXilBGPM1XuTVqx147aFqcKaFLNXXPQyZblFZjcjc1qKYqiV5dCwUZYXlUFXQZ3pOVI
jwSPyaXKkrM0dbROlMklhJKSbG2PiedcoYkytJNQoEcp/D+HP+mz4Pze873sZPnpZpRKFwpDCOgM
zCMa4pzpQwcKAZ+b93oHgoe+ikaDxtApj2Gh/Zjd/N530duxrONH/mJHP70mCiaPtsLcdbVb7/as
obqnf4WGUzpLJPp6vyoRyMiLr/n53abXjt7GTxyOq0fCvB+a0tN7s8hYY3Vpy6u73tXm3dmlnbd/
df52Hr6HpThKCT2eNN8pSYnOkpQ87rp6ntTe/FiTphGJieX5MPW8FjdEn9qWZq/a1ooksYG2DWev
O/0oMqRJnQip5flF0Xojs8XW7IYzK8HPvfxyfJFzImO3i/h6/pWxyYnocWzZvXLoRzj49WVnFh7U
l1wxfLpLrWD5KWlLFa93Z7eZob8vItfrNcpfKEczswk+ySg+hcoPM6kv5Rvb9txayw9MbahvUlpH
M2pcS/JueUpKP3xlhToM3Im0bpylxKOHOloJstFyprRla2nM6HAXvaYvO4EklKmDran44O1RrxZt
jR6nPtPM5mCtmEd6JkYLdc3IW+KQj7qB61TBylxYkyoOYkZlIpcLj4mXVTlcrtbXE7tvmZWR0OOg
3rWs0f9djW6ZRHpJo2iN/1yQjT6uCEfcR7KWZhCPu9ERYl+rsOpTJDRirgoeM92tHk/CbW2G4uf6
h6DjsOMvelFKKTjjbBdWf9/RzKVTIQxmT0IMzP2MwrS90I8BiyOkdzgHHG3tUf5QhH774Piv9KTP
DwJR1o8pI6Em0mKIz0HYxlJc7z58tENTEj3uZcZ6TjZE3lMxVQpTSMOlNFIiRHCfkcKljMcEypOH
Ya3/stb/JAqkR9Ex3WCLBV3a0bPAmjGI8jyiMXeSE4PxeIi6I7HtxIzdK/s8V/I4o53O5hSZCLeL
41qyTQVnLCOXjds3oUNJJb674O5STZjzLERmd5g0JIoWmMRZj/WaEURG6m8cuJiSJSJItm5iTlkY
ySXEeRv+6WiNHgToJRRRE8qkwZypNxouSIpWiS4Y2NCVTvcsDieliKY+BvYEyS1mRWa8pQWFgiuC
UQpkMylyQik3We+tq4aY40ZNx1H9r87Q4lrS92RgpRuQj6Y4RXwOv8KkNfArdBNN5frQ6FG4+J8P
vpjCIeDgkMyn/4u5IpwoSBAbED0A
